api/spec/spec_handler.go

```diff
@@ -576,6 +588,12 @@ func (d *specHandler) SpecOptsFromString(
 	if ok, exportOptions := d.getVal(exportOptionsRegex, str); ok {
 		opts[api.SpecExportOptions] = exportOptions
 	}
+	if ok, mountOptions := d.getVal(mountOptionsRegex, str); ok {
+		// mount options will be provided as a string in the following format
+		// mount_options=k1;k2:v2;k3:v3
+		// convert it to k1,k2=v2,k3=v3
```

Comment:
Why not have the input as how you want it. Is there a problem with the ','

Reply:
Yes the `,` is the issue. This is used by the docker driver and w/o using different symbols, each mount option would end up as a new volume argument.